examples/federated-learning/convert.py

```diff
@@ -57,9 +61,10 @@ def normalize(image, label):
 
 def get_data_from_tfrecord(filename, batch_size: int):
```

Comment:
is thing function used at all? if not it might be better to simply remove it

Reply:
Deleted!